hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/contract/AbstractContractAppendTest.java

```diff
@@ -117,13 +122,9 @@ public void testBuilderAppendToExistingFile() throws Throwable {
   @Test
   public void testAppendMissingTarget() throws Throwable {
     try {
-      FSDataOutputStream out = getFileSystem().append(target);
-      //got here: trouble
-      out.close();
-      fail("expected a failure");
-    } catch (Exception e) {
-      //expected
-      handleExpectedException(e);
+      getFileSystem().append(target).close();
+    } catch (Exception ex) {
+      handleExpectedException(ex);
```

Comment:
again, test is no longer a test of the behavior of any filesystem

Reply:
This should be fixed now after cleaning the commit and revert part in hadoop-common.